kubectl/client.go

```diff
@@ -89,12 +89,18 @@ func (c *Client) Apply(path string, flags ApplyFlags) (string, string, error) {
 	return c.applyPath(path, flags)
 }
 
-// Path returns the filesystem path to the kubectl binary
-func (c *Client) Path() (string, error) {
+// KubectlPath returns the filesystem path to the kubectl binary
+func (c *Client) KubectlPath() (string, error) {
 	kubectlCmd := exec.Command("kubectl")
 	return kubectlCmd.String(), nil
 }
 
+// KustomizePath returns the filesystem path to the kubectl binary
+func (c *Client) KustomizePath() (string, error) {
```

Comment:
Why does this return an error if the error is always nil? (same applies to the other method)

Reply:
Great point! Leftover from when it did (or when I thought it would return an error). Duplicated in the new method.